CommunicationLayer/CommDeviceControl/RadioCommDevice.cpp

```diff
@@ -0,0 +1,55 @@
+#include <QDebug>
+#include <QTimer>
+#include "RadioCommDevice.h"
+
+namespace
+{
+   const QString DEFAULT_PORT_NAME = "/dev/pts/7";
```

Comment:
Ok thanks, I think this will be a consideration I'll add in later when the need arises, at the moment there isn't a need for a different section.